docs/dns-usage.rst

```diff
@@ -0,0 +1,191 @@
+Using the API
+=============
+
+Client
+------
+
+:class:`Client <gcloud.dns.client.Client>` objects provide a means
+configure your DNS applications.  Eash instance holds both a ``project``
+and an authenticated connection to the DNS service.
+
+For an overview of authentication in ``gcloud-python``, see:
+http://gcloud-python.readthedocs.org/en/latest/gcloud-auth.html
```

Comment:
I like the `:doc:` directive better.  If the italics are too much of a bother, we could style them away with CSS.

d8a8b2e